src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -375,6 +375,7 @@ private String getInfo(ContentFilter filter) {
                     break;
                 }
             }
+            result.append(min).append(" Available CPUs:   ").append(Runtime.getRuntime().availableProcessors()).append("\n");
```

Comment:
```suggestion
            result.append(min).append(" Available CPUs:   ").append(runtime.availableProcessors()).append("\n");
```

You have a runtime instance created few lines before :
https://github.com/jenkinsci/support-core-plugin/blob/dbb58348f625dc61c33542c73aafaa8d8f5b5aa7/src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java#L334